models/users.js

```diff
@@ -486,7 +486,7 @@ const getDiscordUsers = async () => {
 const fetchAllUsers = async () => {
   const users = [];
   const usersQuerySnapshot = await userModel.get();
-  usersQuerySnapshot.forEach((user) => users.push(user.data()));
+  usersQuerySnapshot.forEach((user) => users.push({ ...user.data(), id: user.id }));
```

Comment:
don't you think  id property is pushed into the users array twice.

what about below code 
usersQuerySnapshot.forEach((user) => users.push({ ...user.data() }));
return users;